cmd/soroban-rpc/internal/db/migrations/02_transactions.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+
+-- index to find transactions in ledgers by hash
+CREATE TABLE transactions (
+    hash NCHAR(64) PRIMARY KEY,
+    ledger_sequence INTEGER NOT NULL,
```

Comment:
should there be an index on this column so you can trim the transactions table as transactions fall outside the retention window?

Reply:
Good call! I didn't think about this as the column is sorted so it should be easy but obviously sqlite doesn't know better.